vagrant-settings.yaml

```diff
@@ -0,0 +1,45 @@
+# copy it to vagrant-settings.yaml then Vagrantfile
+# will use values from this file
+
+slaves_count: 2
+slaves_ram: 1024
+master_image: solar-master
+master_image_version: null
+slaves_image: fc23
+slaves_image_version: null
+docker_master_image: solar-project/solar-master
+docker_slaves_image: solar-project/solar-master
```

Comment:
We don't support docker three, you can remove all docker stuff from this Vagrantfile.